mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-auth-service-subschema.json

```diff
@@ -23,6 +23,18 @@
                         "tokenRequestInterval": {
                             "description": "Time before token expiration to request a new token",
                             "type": "number"
+                        },
+                        "keyVaultName": {
+                            "description": "Name of key vault to retrieve the certificate from, for authorizing the Service Principal",
+                            "type": "string"
+                        },
+                        "certName": {
+                            "description": "Name of the Service Principal's certificate in the key vault",
+                            "type": "string"
+                        },
+                        "spClientId": {
```

Comment:
Also, @bpkroth would probably want some test JSON configs with and without these keys

Reply:
Yep, they should be at least added to the `tests/config/schemas/services/test-cases/good/full/...` area.

Reply:
Awesome I didnt realize there were those tests supported!
Added good (full & partial) and bad JSON configs to the collection of tests.